nav2_recoveries/plugins/back_up.cpp

```diff
@@ -28,6 +28,7 @@ namespace nav2_recoveries
 BackUp::BackUp()
 : Recovery<BackUpAction>()
 {
+  feedback_ = std::make_shared<BackUpAction::Feedback>();
```

Comment:
list initializer doesn't work when doing `std::make_shared<...>()`? Need to check.

Reply:
What list initializer? Not sure what you mean. This is just a message.

Reply:
Sorry, I mean constructor list initialization.

Reply:
Still not following. What list?

Reply:
I mean this way
```
Backup::Backup()
: Recovery<BackUpAction>(),
  feedback_(std::make_shared<BackUpAction::Feedback>())
```

Reply:
Really not trying to be dense, but I _really_ am failing to understand what list initialization / construction has to do with this. These 2 statements are exactly identical as far as I know. Maybe @naiveHobo understands and I'm just not quite connecting something.

Reply:
I think what Shivang is saying is if it doesn't make a difference, why not move the initialization to the constructor initialization list since that's just a better code practice (which makes sense, I'll do that).

Reply:
Also, initializer list will prevent double initialization. I mean we won't be achieving any thing great in performance wise (may be for this particular case) but will help in making things consistent across the stack with best practice.

Reply:
> list initializer

Ah. I thought you were talking about a list, like the data structure, initalizer, like the bracket initializer. Thank you.

Reply:
Sorry for being unclear. :v: